stdlib/public/SwiftRemoteMirror/SwiftRemoteMirror.cpp

```diff
@@ -866,9 +879,16 @@ swift_async_task_info_t
 swift_reflection_asyncTaskInfo(SwiftReflectionContextRef ContextRef,
                                swift_reflection_ptr_t AsyncTaskPtr) {
   auto Context = ContextRef->nativeContext;
+
+  // Limit the child task and async backtrace iteration to semi-reasonable
+  // numbers to avoid doing excessive work on bad data.
+  unsigned ChildTaskLimit = 1000000;
+  unsigned AsyncBacktraceLimit = 1000;
```

Comment:
Is 1000 large enough here? If this is a back-stop to protected against circularity, maybe 10,000 would be safer? (Or use tortoise-and-hare, which avoids having to set these.)

Reply:
I'm somewhat imitating the crash reporter, which cuts off at 512 (I think) frames in the interest of speed. I believe this is a case where programmers are very likely to be interested in just the top of very deep stacks. I am kind of plucking numbers out of thin air though.